nucliadb_node/src/shards/metadata.rs

```diff
@@ -83,9 +83,12 @@ pub struct ShardMetadata {
     shard_path: PathBuf,
     id: String,
     kbid: String,
-    similarity: Similarity,
     channel: Channel,
+
+    // redundant configuration
+    similarity: Similarity,
     normalize_vectors: bool,
```

Comment:
Shouldn't this be removed?

Reply:
Removing this affects replication and I preferred to do it separately. As replicator creates the shard, it needs all metadata and if it's not stored in shard metadata anymore, we need a way to ask the subindexes for it's metadata.